R/TealDataset.R

```diff
@@ -184,7 +184,7 @@ TealDataset <- R6::R6Class( # nolint
     #' Derive the column labels
     #' @return `character` vector.
     get_column_labels = function() {
-      formatters::var_labels(private$.raw_data, fill = FALSE)
+      teal.data::col_labels(private$.raw_data, fill = FALSE)
```

Comment:
Package always prioritize functions from the package
```suggestion
      col_labels(private$.raw_data, fill = FALSE)
```